networkx/readwrite/gml.py

```diff
@@ -153,12 +153,12 @@ def read_gml(path, label='label', destringizer=None):
 
     label : string, optional
         If not None, the parsed nodes will be renamed according to node
-        attributes indicated by ``label``. Default value: ``'label'``.
+        attributes indicated by `label`. Default value: `'label'`.
```

Comment:
Why are the double backticks important for a literal string?   (I'm not saying they aren't important... I want to understand what the decision process is here.)  In general, the fewer formatting characters the better so the raw text is readable too. But if there's a good reason, then we should use it.